api_v3/services/BaseEntryService.php

```diff
@@ -26,6 +31,47 @@ public function initService($serviceId, $serviceName, $actionName)
             throw new KalturaAPIException(KalturaErrors::ACTION_FORBIDDEN, "anonymousRank");
         }
     }
+	
+	public function adjustArguments(&$arguments = null, $actionParams = null)
+	{
+		$params = requestUtils::getRequestParams();
+		
+		$skipDeserializer = $this->adjustParameters($params);
```

Comment:
having function named adjustParameters and one named adjustArguments is confusing
also the use f this function for adjusting and running dispatcher again at this point is not fully clear